frontend/src/lib/taxonomy.tsx

```diff
@@ -945,6 +977,13 @@ export function getKeyMapping(
             data.description = `${String(data.description)} Data from the first time this user was seen.`
         }
         return data
+    } else if (value in eventToPersonProperties) {
```

Comment:
What I miss here is if this function is called specifically for person properties, otherwise this condition will return true even for event properties, what would I need to do?

Reply:
I probably need to expand the `type` param to accept `TaxonomicFilterGroupType.PersonProperties` as well? or is that something else?

Reply:
`type` being "event" or "element" currently, I'd say adding a "person" option too. Which would be the same as "event", except for this "Latest" prefix. Then we need to pass "person" down in a bunch of places, but that doesn't require any new logic